sections/futures/PositionCard/PositionCard.tsx

```diff
@@ -290,7 +292,7 @@ const PositionCard: React.FC<PositionCardProps> = () => {
 				<DataCol>
 					<InfoRow>
 						<StyledSubtitle>{data.marketShortName}</StyledSubtitle>
-						<StyledValue>{data.marketPrice}</StyledValue>
+						<StyledValue className={marketPriceColor}>{data.marketPrice}</StyledValue>
```

Comment:
If we create a shared `PriceColor` component like mentioned above could we just use that here instead?